### ISSUE TITLE
Basic Authentication

### ISSUE THREAD

**Issue by schmidwi** [usebruno/bruno#286]
Add the ability to pass basic authentication via username / password (ideally from environment variables).

**Comment by n00o** [comment]
Bruno has an internal JavaScript library called `btoa` that allows you to encode base64 characters which can be used for doing your own basic authentication. Use the basic format below inside of the Script > Pre Request section:
```JavaScript
const btoa = require("btoa");
var  creds = btoa(bru.getEnvVar("username")+":"+bru.getEnvVar("password"));
req.setHeader("Authorization","Basic "+creds);
```
You will have to still need the username and password inside the environment.

**Comment by lared** [comment]
It's already implemented.

**Comment by helloanoop** [comment]
This is available in the latest release `0.19.0`

**Comment by Mnior** [comment]
`btoa` is not correctly work with non ASCII symbols
